scripts/weblate/double_languages.py

```diff
@@ -1,45 +1,47 @@
-
-
 import wlc
 import apikey
 import requests
 import math
-import json
 from requests.exceptions import HTTPError
+import json
 
 
 w=wlc.Weblate(apikey.myAPIKey)
 projects={'mobile':{},'server':{},'webapp':{},'glossary':{}}
-projects['mobile']['shipped']='mattermost-mobile_master'
+projects['mobile']['shipped']='mattermost-mobile-v2'
 projects['mobile']['wip']='mattermost-mobile-wip'
 projects['server']['shipped']='mattermost-server_master'
 projects['server']['wip']='mattermost-server-wip'
 projects['webapp']['shipped']='mattermost-webapp_master'
 projects['webapp']['wip']='mattermost-webapp-wip'
 projects['glossary']['shipped']='glossary'
 projects['glossary']['wip']='glossary'
+
 current_notification_state={"state":"false"}
 previous_notification_state= json.load(open("./notification_state.txt"))
 
 for project in projects:
   shippedLanguages={}
   WIPLanguages={}
-### GETTING THE SHIPPED LANGUAGES ###
-  #print('GETTING SHIPPED LANGUAGES FOR '+project)
-  shippedProjects=w.get('https://translate.mattermost.com/api/components/mattermost/'+projects[project]['shipped']+'/translations/')
-  for shippedLanguage in shippedProjects['results']:
-    if shippedLanguage['language']['code']=='en':
-      continue
-    shippedLanguageCode=shippedLanguage['language']['code']
-    shippedLanguageName=shippedLanguage['language']['name']
-    shippedLanguages[shippedLanguageCode]=shippedLanguageName
+  page=1
+  next='https://translate.mattermost.com/api/components/mattermost/'+projects[project]['shipped']+'/translations/'
+  shippedProjects=w.get(next)
+  lastpage=math.ceil(shippedProjects['count']/20)  
```

Comment:
Instead of hard-coding the assumption that 20 results come back per page, would it make sense to rely on [weblate's `next` property](https://docs.weblate.org/en/latest/api.html#authentication-and-generic-parameters) in the response, giving the exact URL to use next?

Reply:
10%% true! I fixed this